src/services/utilities.ts

```diff
@@ -4288,3 +4302,93 @@ export function isBlockLike(node: Node): node is BlockLike {
             return false;
     }
 }
+
+/** @internal */
+export function getTargetFileImportsAndAddExportInOldFile(

```

Comment:
Maybe `src/services/refactors/helpers.ts` would be a better place for this, judging by the number of new imports from `ts.refactor`.